tests/test_process.py

```diff
@@ -108,6 +111,157 @@ def test_load_rain_file(rain_data_file):
     ]
 
 
+def test_load_rain_file_flanders(rain_data_file_flanders):
```

Comment:
not that many of the things you test are handled by https://github.com/watem-sedem/rfactor/blob/68e68a454af177ae5445be8d50a7f2a23545676f/tests/test_process.py#L82, so you shuldn't repeat those tests.